src/Controls/StreetControls.js

```diff
@@ -63,6 +79,26 @@ class StreetControls extends FirstPersonControls {
         this.keyGoToNextPosition = 90;
         this.keyGoToPreviousPosition = 83;
         this.keySetCameraToCurrentPosition = 65;
+
+        // Raycaster used to pick objects
+        this.raycaster = new THREE.Raycaster();
```

Comment:
could you remove `raycaster`, it's useless

Reply:
yep